src/OpenDiffix.Core/Anonymizer.fs

```diff
@@ -274,13 +278,13 @@ let countDistinct
   // we can only report the count of values we already know to be safe as they
   // individually passed low count filtering.
   if byAid |> List.exists ((=) None) then
-    if safeCount > 0L then Integer safeCount else Null
+    if safeCount > 0L then safeCount |> Result.Count else NotEnoughAIDVs
```

Comment:
Pipe can be removed so that it reads more straightforward: `Result.Count safeCount`.